docs/changelogs/vNEXT.rst

```diff
@@ -55,6 +55,13 @@ Note that custom constructors are still supported for the Rust library since
 specifying a static ``new`` method for a type is nothing more than a
 convention.
 
+Improvements to the debugger
+----------------------------
+Three ways to step through queries, step, over and out.
```

Comment:
None of these commands are new, but the implementation is. Maybe add a blurb about how each has changed (+ optionally link to the updated documentation)?